index.html

```diff
@@ -0,0 +1,898 @@
+          </ul>
+        </div>
+        <div class="section-two-item">
+          <div class="section-two-item4 img-size-1 no-repeat">
+
+          </div>
+          <h4 class="story-title">
+            <a href="#">Google Sheets can improve your app with machine learning here's how </a>
+          </h4>
+          <ul class="story-title">
+            <li class="color-gray"><a class="color-gray" href="#">Dale Markowitz</a> - <div> 4 hours ago</div></li>
+          </ul>
+        </div>
+        <div class="section-two-item">
+          <div class="section-two-item5 img-size-1 no-repeat">
+
+          </div>
+          <h4 class="story-title">
+            <a href="#">How to come up with the perfect content idea </a>
+          </h4>
+          <ul class="story-title">
+            <li class="color-gray"><a class="color-gray" href="#">The Conversation</a> - <div>4 hours ago</div></li>
+          </ul>
+        </div>
+        <div class="section-two-item">
+            <div class="section-two-item6 img-size-1 no-repeat">
+
+            </div>
+            <h4 class="story-title">
+              <a href="#">Gates Foundation sold $1B worth of Berkshire stock last quarter it's still up $500M </a>
+            </h4>
+            <ul class="story-title">
+              <li class="color-gray"><a class="color-gray" href="#">David Canellis</a> -  <div>10 hours ago</div></li>
+            </ul>
+        </div>
+        <div class="section-two-item">
+            <div class="section-two-item7 img-size-1 no-repeat">
+
+            </div>
+            <h4 class="story-title">
+              <a href="#">India finally restores 4G in some parts of Kashmir after 376 days</a>
+            </h4>
+            <ul  class="story-title">
+              <li class="color-gray"><a class="color-gray" href="#">Ivan Mehta </a> - <div>4 hours ago</div></li>
+            </ul>
+        </div>
+        <div class="section-two-item">
+            <div class="section-two-item8 img-size-1 no-repeat">
+
+            </div>
+            <h4 class="story-title">
+              <a href="#">Pardon the Intrusion #24: The clock is TikToking</a>
+            </h4>
+            <ul class="story-title">
+              <li class="color-gray"><a class="color-gray" href="#">Ravie Lakshmanan</a> - <div>4 hours ago</div></li>
+            </ul>
+        </div>
+      </div>
+      </Section>
+      
+  <!-- section three -->
+        <section class="section-three  center ">
+            <div class="section-three-title-flex center list ">
+              <div class="titles ">
+                <a href="#">Latest funding rounds on index.co <i class="fas fa-angle-right"></i></a>
+              </div>
+              <div class="right-title">
+                <a href="#" class="list-b">Add info</a>
+                <a href="#" class="list-b">Get update by email</a>
+                <a href="#" class="list-b last-a" >See more</a>
+              </div>
```

Comment:
on large screens: 

- [ ] the small titles should be on the right in one line

| Yours | Original |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/11758151/92156977-ac48b900-ee21-11ea-841a-1012a64df9e2.png) | ![image](https://user-images.githubusercontent.com/11758151/92157018-bc609880-ee21-11ea-8045-0d1a2b6c38e7.png) |

on medium screens: 

- [ ] the small titles should on the same line under the big title, with borders in between

| Yours | Original |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/11758151/92157168-f7fb6280-ee21-11ea-9b4f-0898a264fd76.png) | ![image](https://user-images.githubusercontent.com/11758151/92157203-06497e80-ee22-11ea-8d18-731cf55cd697.png) |